tests/python/gpu/test_operator_gpu.py

```diff
@@ -1996,6 +1996,31 @@ def get_num_devices():
         _check_batchnorm_result(mx.nd.random.uniform(shape=(4, 1, 4, 4)),
                                 num_devices=ndev, cuda=True)
 
+def _check_inplace_abn(input, layer1, layer2):
+    with mx.autograd.record():
+        output1 = layer1(input)
+        output2 = layer2(input)
+        loss1 = (output1 ** 2).sum()
+        loss2 = (output2 ** 2).sum()
+        mx.autograd.backward(loss1)
+        mx.autograd.backward(loss2)
+    assert_almost_equal(loss1.asnumpy(), loss2.asnumpy(), atol=1e-3, rtol=1e-3)
+    assert_almost_equal(output1.asnumpy(), output2.asnumpy(), atol=1e-3, rtol=1e-3)
+    assert_almost_equal(input1.grad.asnumpy(), input2.grad.asnumpy(), atol=1e-3, rtol=1e-3)
+
+def test_inpabn():
```

Comment:
This doesn't seem the right place to put this test. For this PR, it would be better to have:
1. An operator test in https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_contrib_operator.py
2. A gluon block test in https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_gluon_contrib.py

Reply:
This will be a gpu test. Still working in progress.

Reply:
I see. Nonetheless, given the scope of the change the suggested tests should be appropriate.